api/src/main/java/ca/bc/gov/educ/api/gradalgorithm/service/ParallelDataFetch.java

```diff
@@ -23,11 +23,15 @@ public class ParallelDataFetch {
     @Autowired
```

Comment:
Logger not used and is referencing GradCourseService.class. Please remove if not used or update to use ParallelDataFetch.class